lib/icinga/command.ti

```diff
@@ -28,6 +28,8 @@ namespace icinga
 abstract class Command : CustomVarObject
 {
 	[config] Value command (CommandLine);
+	[config] Value plugin;
```

Comment:
Is there ever a case where `plugin` is something other than a string? What's the rationale for using `Value` here? Nullability checks?